app/views/api/v2/models/main.json.rabl

```diff
@@ -3,3 +3,9 @@ object @model
 extends "api/v2/models/base"
 
 attributes :info, :created_at, :updated_at, :vendor_class, :hardware_model
+
+if params.has_key?(:spa)
```

Comment:
I think we can show the host count in every mode. That should not hurt. Also, I'd consider this more of a feature flag. Something like `show_permissions` vs `spa`. Then it's automatically clear what you gain if you set the parameter. We don't need a new concept for this imho.

Reply:
Thanks a lot @timogoebel for your help.